src/main/java/com/thehrdivision/entities/Employee.java

```diff
@@ -0,0 +1,52 @@
+package com.thehrdivision.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+public class Employee {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private String firstname;
+    private String lastname;
+    private String gender;
+    private LocalDate dateOfBirth;
+    private LocalDate dateJoined;
+    private LocalDate dateLeft;
+    private Boolean disability;
+
+    @OneToOne
+    @JoinColumn(name = "address_id")
+    private Address address;
+
+    @OneToOne
+    @JoinColumn(name = "user_id")
+    private Person user;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<TrainingHistory> trainingHistory;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<PerformanceReview> performanceReviews;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<Payroll> payrolls;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<EmployeeSkills> employeeSkills;
+
+    @OneToMany(mappedBy = "employee")
+    private Set<JobAssigments> jobAssigments;
+
+    // TODO: 19.03.2022 Employee <--> Benefit relation
```

Comment:
If we have another entity please create a task in JIRA, and delete this comment.